zt_backend/runner/execute_code.py

```diff
@@ -82,7 +82,7 @@ def execute_request(request: request.Request, state: UserState):
             downstream_cells = [cell.id for cell in request.cells if cell.cellType in ['code', 'sql']]
         for code_cell_id in downstream_cells:
             code_cell = dependency_graph.cells[code_cell_id]
-            if code_cell_id!=request.originId and code_cell.nonReactive:
+            if request.originId and code_cell_id != request.originId and code_cell.nonReactive:
```

Comment:
Can you explain this one to me? I believe there was already handling here for an empty originId?

Reply:
Not require, I just mentioned it explicitly. Removed it.